### PR TITLE
Fix mnuMethodOrNilFor: for method wrappers

### PR DESCRIPTION
**Pull request by guillep**
Method wrappers should not be compiled

### CODE REVIEW

File: smalltalksrc/VMMakerTests/VMImageHeaderWritingTest.class.st [hunk 1]
```diff
@@ -167,6 +157,16 @@ VMImageHeaderWritingTest >> testWritingImageWritesCorrectImageFormat [
...
+{ #category : #tests }
+VMImageHeaderWritingTest >> testWritingImageWritesCorrectImageVersion [
+
+	| header |
+
+	header := self readHeader.
+
+	self assert: header imageVersion equals: interpreter getImageVersion
+]
+
```
Comment (PalumboN) [inline smalltalksrc/VMMakerTests/VMImageHeaderWritingTest.class.st]: ??

File: smalltalksrc/VMMakerTests/VMJittedLookupTest.class.st [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+	
+	"Install the selector as a DNU selector"
+	memory splObj: SelectorDoesNotUnderstand put: selectorOop.
+	
+	foundMethod := interpreter mnuMethodOrNilFor: receiver.
+	self assert: foundMethod equals: methodOop
+]
+
+{ #category : #tests }
+VMJittedLookupTest >> testLookUpMNUWithWrapperShouldNotJItCompile [
```
Comment (PalumboN) [inline smalltalksrc/VMMakerTests/VMJittedLookupTest.class.st]:
```suggestion
VMJittedLookupTest >> testLookUpMNUWithAnyNonMethodObjectShouldNotJItCompile [
```

**Review by PalumboN — APPROVED** [review]

File: smalltalksrc/VMMakerTests/VMImageHeaderWritingTest.class.st (hunk 1 above)
Comment (tesonep) [reply smalltalksrc/VMMakerTests/VMImageHeaderWritingTest.class.st]: I added that test, in the setup sets a value to check (in the interpreter) and that should be preserved in the header

**Comment by guillep** [comment]
tests are green!

**Pull request merged**